config.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  path:
+    type: string
+    default: ""
+    description: >
+      The organization or the repository to register the self-hosted runners. For repository the
+      path should be in the "{owner}/{repo}" format.
+  group:
+    type: string
+    default: "default"
+    description: >
+      The organization runner group to register the self-hosted runner under. This has no effect on
+      runners under a repository.
+  token:
+    type: string
+    default: ""
+    description: The GitHub Personal Access Token for registering the self-hosted runners.
+  virtual-machines:
+    type: int
+    default: 0
```

Comment:
Why not 1?

Reply:
This can be changed to 1, which is likely a more user friendly setting for new users.

Reply:
Minor comments